.github/workflows/ci.yml

```diff
@@ -58,9 +58,15 @@ jobs:
       with:
         command: build
         args: --features webview --example webview_custom_protocol
-    # fails because it needs uikit, which is not
-    # building for me
-    #- uses: actions-rs/cargo@v1
-    #  with:
-    #    command: build
-    #    args: --example ios-beta
+    # Since it's all Objective-C message passing under the hood, we're
+    # really just looking for whether we've broken the iOS build. It is likely
+    # that more robust tests/checking infrastructure should exist for this side
+    # of things as the iOS portion gets iterated on.
+    #
+    # (e.g, this at the moment will not catch invalid selector calls, like if an appkit-specific
+    # selector is used for something on iOS)
+    - uses: actions-rs/cargo@v1
+      with:
+        command: build
+        target: x86_64-apple-ios
+        args: --example ios-beta --no-default-features --features uikit,autolayout
```

Comment:
From what I can tell, the build error you got in CI is the same as what I got when I ran `cargo build --example ios-beta --no-default-features --features uikit,autolayout` (notice the lack of `--target x86_64-apple-ios`).